fli-iam/shanoir-ng
# Shanoir issue#1013 Allow file save on preclinical examination (blood gaz data & physiological data)

How to test ?
- Import preclinical data and create a new preclinical examination
- Add to this examination some data on blood gas data & physiological data.
- Check that you can still download this data after the import is finished
